.dockerignore

```diff
@@ -0,0 +1,15 @@
+#Ignoring git and cache folders
```

Comment:
This file is used to ignore files sent into the docker daemon's "context". https://docs.docker.com/engine/reference/builder/#dockerignore-file

Used to ignore files/paths during our dockerfile's COPY command